src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import Courses from '../src/components/Courses/Courses';
+import Header from './components/Header/Header';
+import { mockCourses } from './data/data';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+
+function App() {
```

Comment:
For functional components (we will call them _FC_ in future) please, use **arrow functions** (AF) approach.
Take a look at JS example [here](https://www.w3schools.com/js/js_arrow_function.asp) and React example [here](https://www.w3schools.com/react/react_es6_arrow.asp)